test/framework/src/main/java/org/opensearch/test/engine/MockEngineFactory.java

```diff
@@ -46,6 +47,6 @@ public MockEngineFactory(Class<? extends FilterDirectoryReader> wrapper) {
 
     @Override
     public Engine newReadWriteEngine(EngineConfig config) {
```

Comment:
A comment explaining why this change is needed could be useful for contributors without segrep knowledge.

Reply:
Sure